packages/cli-plugin-metro/launchPackager.command

```diff
@@ -0,0 +1,8 @@
+THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")")" && pwd)
+
+. $THIS_DIR/launchPackager.sh
```

Comment:
Is there a reason for why we need to call `launchPackager.sh`? Can we merge these files?

Reply:
Yup, I checked it and if I change permission to `launchPackager.sh` the `launchPackager.command` file is useless.